integrationTests/cypress/integration/redmine_administration.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Administrative Procedures
+
+  @requires_testuser
+  Scenario: cas user + no redmine user => login => create normal redmine account
```

Comment:
In integration tests for other dogus, we have always used prose for scenario description. Therefore, I would suggest wording the descriptions so that they are easier to understand for the willing reader.

Maybe something like:
" cas user + no redmine user => login => create normal redmine account" --> "An ordinary CAS user logs into redmine for the first time and a normal redmine account is created"